docs/osquery/osquery.asciidoc

```diff
@@ -3,13 +3,13 @@
 [[osquery]]
 = Osquery
 
-https://osquery.io[Osquery] is an open source tool that lets you query operating systems, like a database, providing you with visibility into your infrastructure and operating systems.
+https://osquery.io[Osquery] is an open source tool that lets you query operating systems like a database, providing you with visibility into your infrastructure and operating systems.
 Using basic SQL commands, you can ask questions about devices, such as servers, 
 Docker containers, and computers running Linux, macOS, or Windows.
 The https://osquery.io/schema[extensive schema] helps with a variety of use cases,
 including vulnerability detection, compliance monitoring, incident investigations, and more.  
 
-With Osquery, you can:
+With Osquery in {kib}, you can:
 
  * Run live queries for one or more agents
  * Schedule queries to capture changes to OS state over time
```

Comment:
Should we mention query packs here?

Reply:
Good suggestion! It now specifies `Schedule query packs` instead of just `Schedule queries`.